src/compass/utils/age.py

```diff
@@ -282,6 +285,9 @@ def find_peak(cslc_file, x_loc, y_loc, mission_id='S1',
     # Extract an area around x_loc, y_loc
     img = arr[upperleft_y:lowerright_y, upperleft_x:lowerright_x]
 
+    # Check if the SNR of the peak is above the threshold
+    snr_peak_db = get_snr_peak(img)
```

Comment:
```suggestion
    snr_peak_db = get_snr_cr(img)
```

For the same reason as above